src/components/ShowMenus/ShowMenus.js

```diff
@@ -7,13 +8,15 @@ export default function ShowMenus({menus, curClickedCategory}) {
         .filter(menu => curClickedCategory === 'All' || menu.category === curClickedCategory)
         .map(menu => {
           return (
-            <Menu key={menu.id}>
-              <MenuImg src={menu.image} alt={menu.name} />
-              <MenuNamePriceContainer>
-                <Menuname>{menu.name}</Menuname>
-                <Menuprice>{menu.price + '€'}</Menuprice>
-              </MenuNamePriceContainer>
-            </Menu>
+            <Link key={menu.id} to={`/details/${String(menu.id)}`}>
+              <Menu key={menu.id}>
```

Comment:
You don't need this `key` here. One is enough :) 
```suggestion
              <Menu>
```

Reply:
You're right, this is from version before Link comes in, forget to delete it.